stisim/plotting.py

```diff
@@ -0,0 +1,84 @@
+####### PLOT RESULTS
+import matplotlib.pyplot as plt
+import numpy as np
+import stisim as ss
+from stisim import interventions as ssi
+import sciris as sc
+
+
+def plot_cum_infections(sim, ax, disease):
```

Comment:
If desired, these functions could be refactored to also take an argument for the result being plotted, so they could be combined into a single `plot_result()` function. Would be nice if `Result` objects had a `label` attribute though -- added an issue here: https://github.com/starsimhub/starsim/issues/510

Reply:
Made an issue for this